p2p_wallet/Scenes/Main/Swap/Swap/Subviews/Input/SwapInputView.swift

```diff
@@ -86,13 +86,16 @@ private extension SwapInputView {
     }
 
     var amountField: some View {
-        DecimalTextField(value: $viewModel.amount, isFirstResponder: $viewModel.isFirstResponder, textColor: $viewModel.amountTextColor) { textField in
+        AmountTextField(
+            value: $viewModel.amount,
+            isFirstResponder: $viewModel.isFirstResponder,
+            textColor: $viewModel.amountTextColor,
+            maxFractionDigits: Int(viewModel.token.token.decimals),
+            decimalSeparator: "."
+        ) { textField in
             textField.font = .font(of: .title1)
-            textField.keyboardType = .decimalPad
             textField.isEnabled = viewModel.isEditable
             textField.placeholder = "0"
-            textField.maximumFractionDigits = Int(viewModel.token.token.decimals)
```

Comment:
The limit of decimals number after comma is missing, for example: USDC has decimals of 6, so the minimum amount should be 0.000001, not 0.000000001
<img width="378" alt="image" src="https://user-images.githubusercontent.com/6975538/226269791-ae9331fd-d8ea-4ca5-9dd2-38c2bb446694.png">